certified-connectors/Zellis/apiProperties.json

```diff
@@ -42,13 +42,10 @@
       },
       "token:ClientId": {
         "type": "string",
-        "metadata": {
-          "sourceType": "AzureActiveDirectoryTenant"
-        },
         "uiDefinition": {
           "constraints": {
             "required": "true",
-            "hidden": "false"
+            "hidden": false
```

Comment:
@SaiChander-MS @vmanoharas This is latest commit and review comments are implmented